examples/sample/release.yaml

```diff
@@ -31,5 +31,28 @@ spec:
 #         name: svals
 #         namespace: wordpress
 #         optional: false
+#  connectionDetails:
+#    - apiVersion: v1
+#      kind: Service
+#      name: wordpress-example
+#      namespace: wordpress
+#      fieldPath: spec.clusterIP
+#      #fieldPath: status.loadBalancer.ingress[0].ip
+#      toConnectionSecretKey: ip
+#    - apiVersion: v1
+#      kind: Service
+#      name: wordpress-example
+#      namespace: wordpress
+#      fieldPath: spec.ports[0].port
+#      toConnectionSecretKey: port
+#    - apiVersion: v1
+#      kind: Secret
+#      name: wordpress-example
+#      namespace: wordpress
+#      fieldPath: data.wordpress-password
+#      toConnectionSecretKey: password
```

Comment:
I have a `ServiceAccountKey.cloudplatform.gcp.upbound.io/v1beta1` inside a helm chart, which creates a connection secret at `<uid>-serviceaccountkey` in the `crossplane` namespace.

So I made a list entry like this on my `Release`:
```
            - apiVersion: v1
              kind: Secret
              name: ""
              namespace: crossplane
```
And then I'm patching the name like so:
```
        - type: CombineFromComposite
          combine:
            variables:
              - fromFieldPath: metadata.uid
            strategy: string
            string:
              fmt: "%s-serviceaccountkey"
          toFieldPath: spec.connectionDetails[0].name
          policy:
            fromFieldPath: Required
```

Now the issue is that on the `Release` I get an event that says:
```
│   Warning  CannotObserveExternalResource  31s (x5 over 3m34s)    managed/release.helm.crossplane.io  cannot get connection details: object is not part of release: {Secret crossplane 8578b338-7520-44e8-a3e7-a1555abfd257-serviceaccountkey  v1  }
```
Which is of course true, because the connection secret is not part of that release, it's just a result of it.
But the connection secret exists:
```
╰─ kg secret -n crossplane 8578b338-7520-44e8-a3e7-a1555abfd257-serviceaccountkey
NAME                                                     TYPE                                DATA   AGE
8578b338-7520-44e8-a3e7-a1555abfd257-serviceaccountkey   connection.crossplane.io/v1alpha1   2      7m33s
```

How can I propagate the secret of that `Secret` resource outside of the release? I thought it should be possible since I'm just referring to an arbitrary secret by name and namespace that the controller has access to...

Reply:
As we discussed offline, `skipPartOfReleaseCheck: true` should do the trick.